packages/devtools_app/web/index.html

```diff
@@ -55,134 +26,20 @@
-                <span>
-                    <span class="optional-1140">Performance</span>
-                </span>
-            </a>
-            <a disabled>
-                <span class="octicon octicon-package"></span>
-                <span>
-                    <span class="optional-1140">Memory</span>
-                </span>
-            </a>
-            <a disabled>
-                <span class="octicon octicon-bug"></span>
-                <span>
-                    <span class="optional-1140">Debugger</span>
-                </span>
-            </a>
-            <a disabled>
-                <span class="octicon octicon-clippy"></span>
-                <span>
-                    <span class="optional-1140">Logging</span>
-                </span>
-            </a>
-        </nav>
-
-        <div id="global-actions" class="masthead-nav"></div>
-
-        <div class="masthead-nav">
-            <span horiz-padding class="masthead-item">•</span>
-            <div class="masthead-item action-button active" id="icon-nav">
-                <span class="octicon octicon-info"></span>
-            </div>
-            <div class="masthead-item action-button active" id="send-feedback-button" title="Send Feedback">
-                <span class="octicon octicon-comment"></span>
-            </div>
-            <span horiz-padding class="masthead-item">•</span>
-            <div hidden="true" class="masthead-item action-button active" id="try-flutter-web-devtools" title="Try the alpha version of DevTools &#013;written in Flutter!">
-                <span class="octicon octicon-beaker"></span>
-            </div>
-        </div>
-    </header>
-
-    <div id="messages-container" class="container legacy-dart"></div>
-
-    <form id="ga-dialog" class="container legacy-dart"></form>
-    <form id="connect-dialog" class="container legacy-dart"></form>
-    <form id="snapshot-message" class="container legacy-dart"></form>
-
-    <div id="content" class="container legacy-dart" flex layout vertical style="overflow: auto;">
-        <div class="blankslate">
-            <p>Loading...</p>
-        </div>
-    </div>
-
-    <footer flex layout horizontal none class="footer align-items-center legacy-dart">
-        <div class="footer-status" id="global-status" flex>
-            <select class="form-select select-sm" disabled></select>
-        </div>
-
-        <div class="footer-status text-center" id="page-status" flex horiz-padding>
-        </div>
-
-        <div class="footer-status text-right" id="auxiliary-status" flex>
-            &nbsp;
-        </div>
-    </footer>
-
-    <div id="toast-container" class="legacy-dart">
-    </div>
+<body>
+  <!-- This script installs service_worker.js to provide PWA functionality to
+       application. For more information, see:
+       https://developers.google.com/web/fundamentals/primers/service-workers -->
+  <script>
+    if ('serviceWorker' in navigator) {
```

Comment:
I don't know that we need this, but we don't need to resolve for this PR. We should file an issue to investigate whether we want to use a service worker w/ DevTools.

Reply:
This was copied from the default configuration for a flutter web app. Filed issue to investigate further https://github.com/flutter/devtools/issues/1963